src/routes/JoinChallange.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import { useNavigate } from 'react-router-dom';
+import { Flex} from "antd";
+import Heading from "../components/Heading";
+import CustomButton from "../components/CustomButton";
+import Paragraph from "../components/Paragraph";
+import {CustomInput} from "../components/CustomInputs";
+import Caption from "../components/Caption";
+
+
+const JoinChallange = () => {
+    const navigate = useNavigate();
+
+    const [phoneNumber, setPhoneNumber] = useState('');
+
+    const isButtonDisabled = !phoneNumber;
+
+    const handlePhoneNumberChange = (e) => {
+        setPhoneNumber(e.target.value);
+    };
+
+    const handleJoin = () => {
+        navigate('/Join-challange-success');
+    }
+
+    const handleSkip = () => {
+        navigate('/menu', { state: { skipModal: true } });
```

Comment:
My understanding is that we should have modal within menu page instead of navigating to another route? same from JoinChallengeSuccess

Reply:
resolved in meeting